test/Foreign/Marshal/Alloc/CompatSpec.hs

```diff
@@ -0,0 +1,27 @@
+module Foreign.Marshal.Alloc.CompatSpec (main, spec) where
+
+import           Test.Hspec
+import           Test.QuickCheck
+
+import           Foreign.Marshal.Alloc.Compat
+import           Foreign.Marshal.Utils
+import           Foreign.Ptr
+import           Foreign.Storable.Compat
+
+main :: IO ()
+main = hspec spec
+
+spec :: Spec
+spec = do
+  describe "calloc" $
+    it "allocates memory with bytes of value zero" $
+      property $ do
```

Comment:
Maybe I'm missing something obvious here, this property is not parameterized.  Does it make any difference to run it once vs. 100 times?